src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_IndexerAccess.cs

```diff
@@ -432,6 +432,24 @@ private enum PatternIndexOffsetLoweringStrategy
                 strategy = PatternIndexOffsetLoweringStrategy.UseAsIs;
                 return VisitExpression(operand);
             }
+            else if (unloweredExpr is BoundObjectCreationExpression { Constructor: MethodSymbol constructor, Arguments: { Length: 2 } arguments, ArgsToParamsOpt: { IsDefaultOrEmpty: true }, InitializerExpressionOpt: null } &&
```

Comment:
>InitializerExpressionOpt: null [](http://example.com/codeflow?start=190&length=30)

nit: do we have a indexing test with an Index initialization with an initializer?

Reply:
> do we have a indexing test with an Index initialization with an initializer?

No, the type doesn't have any member that can be initialized, but I decided to have the check for completeness.